opm/core/props/phaseUsageFromDeck.hpp

```diff
@@ -119,6 +119,15 @@ namespace Opm
         else
             pu.phase_pos[BlackoilPhases::PolymerMW] = -1;
 
+        // Add foam info
+        pu.has_foam = phases.active(Phase::FOAM);
+        if (pu.has_foam) {
+            pu.phase_pos[BlackoilPhases::Foam] = numActivePhases;
```

Comment:
= numActivePhases++;

Reply:
I prefer the separate line, and that is how it is done for the existing components, so I'll keep this as is.